zone/perl_npc.cpp

```diff
@@ -204,6 +204,7 @@ bool Perl_NPC_IsOnHatelist(NPC* self, Mob* mob) // @categories Hate and Aggro
 void Perl_NPC_RemoveFromHateList(NPC* self, Mob* mob) // @categories Hate and Aggro
 {
 	self->RemoveFromHateList(mob);
+	self->RemoveFromRampageList(mob);
```

Comment:
I see you did Perl but not Lua?

Reply:
There is not a lua equivalent.  Lua has `RemoveFromHateLists` which has that built in perl has the singular `RemoveFromHateList`